modules/kubernetes-addons/main.tf

```diff
@@ -68,6 +68,14 @@ module "argocd" {
   add_on_config       = { for k, v in local.argocd_add_on_config : k => v if v != null }
 }
 
+module "argo_rollouts" {
+  count             = var.enable_argo_rollouts ? 1 : 0
+  source            = "./argo-rollouts"
+  eks_cluster_id    = var.eks_cluster_id
+  helm_config       = var.argo_rollouts_helm_config
```

Comment:
Can you please pass tags here? I forgot to add them in my PR.

Reply:
done